### PR TITLE
[Add] Movie detail UI

### PR DESCRIPTION
**Pull request by ghost**
## What's this PR do ?

- Implement Movie detail UI

## Checklist

- [ ] Write/Update Unit Test (if have)
- [ ] Fill link PR into ticket and the opposite
- [ ] Validate UI/Model/API with latest specs
- [x] Add UI pics to this PR (if have).

## Library

_(List library, third party add new, include version)_

- Library: AppCompat v1.2.0

## Impacted Areas in Application

_(List features, api, models or services that this PR will affect)_

- foo/bar

## Demo

https://github.com/namnh-0652/flutter_movie_clean/assets/85486860/a59fec3d-a2bb-4c6e-ac7f-6bff62337a12

### CODE REVIEW

File: lib/pages/login/login.dart [hunk 1]
```diff
@@ -132,12 +132,8 @@ class _LoginPageState extends State<LoginPage> {
         title: context.l10n.login,
         textStyle: TextStyle(fontSize: 18.sp, color: AppColors.white),
         onPressed: () => {
-          if (_formKey.currentState!.validate())
-            {
-              // TODO: call login here
-              context.go(MainPage.routeLocation,
-                  extra: {"user": _emailTextController.text})
-            }
+          context.go(MainPage.routeLocation,
```
Comment (namnh-0652) [inline lib/pages/login/login.dart]: revert the validation

File: lib/pages/moviedetail/movie_detail_page.dart [hunk 2]
```diff
@@ -1,17 +1,295 @@
...
+            _buildTabContent()
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildTopPage(BuildContext buildContext) {
+    return SizedBox(
+      height: MediaQuery.of(buildContext).size.height * 0.6,
```
Comment (namnh-0652) [inline lib/pages/moviedetail/movie_detail_page.dart]: try `0.6.sh`, `1.sw`

File: lib/route/app_router.dart [hunk 3]
```diff
@@ -27,14 +28,6 @@ final appRouter = GoRouter(
     GoRoute(
       path: MainPage.routeLocation,
       name: MainPage.routeName,
-      redirect: (context, state) {
```
Comment (namnh-0652) [inline lib/route/app_router.dart:30]: revert this

File: lib/pages/moviedetail/components/tab_item.dart [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import '../../../gen/colors.gen.dart';
+
+class TabItem extends StatelessWidget {
```
Comment (namnh-0652) [inline lib/pages/moviedetail/components/tab_item.dart:5]: is this used?

Comment (ghost) [reply lib/pages/moviedetail/components/tab_item.dart:5]: Yes. Because the cast and more tabs have the same UI

File: assets/images/ic_play_white.svg
Comment (truyentd-2427) [inline assets/images/ic_play_white.svg:1]: looks like icon is not circular

**Review by namnh-0652 — APPROVED** [review]

**Pull request merged**